### PR TITLE
Fix/update readme

### LINKED ISSUE #123: React Class Components vs Functional Components w/hooks
Title says it all. What's our opinion on using class components vs functional components with hooks?

### PR DESCRIPTION
**Pull request by xarielx**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Closes #123 

Modified Readme to include the use of utilizing functional components with hooks in react, as well as a link to the introduction of these in the [React page.](https://reactjs.org/docs/hooks-intro.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ To better communicate and more easily build an API and UI, we've decided on a co
...
 We are planning to use the following tools:
 
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
-- [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
-- [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
-- A [React](https://reactjs.org/) frontend using JavaScript/TypeScript and CSS (not Sass)
-- [Next.js](https://nextjs.org/) for server-side rendering
+* [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
+* [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
+*  [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
* [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
```

File: README.md [hunk 2]
```diff
@@ -29,11 +29,12 @@ To better communicate and more easily build an API and UI, we've decided on a co
...
+*  A [React](https://reactjs.org/) frontend using JavaScript/TypeScript and CSS (not Sass)
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
* A [React](https://reactjs.org/) frontend using JavaScript/TypeScript and CSS (not Sass)
```

File: README.md [hunk 3]
```diff
@@ -29,11 +29,12 @@ To better communicate and more easily build an API and UI, we've decided on a co
...
+    * Utilizing Functional Components with [Hooks](https://reactjs.org/docs/hooks-intro.html)
+*  [Next.js](https://nextjs.org/) for server-side rendering
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
* [Next.js](https://nextjs.org/) for server-side rendering
```

**Comment by allella** [comment]
Since, there's 1 bullet point, how about we convert this to bullets for each technology, like

A React frontend using:
- JavaScript/TypeScript
- CSS (not Sass)
- Functional Components with Hooks

**Comment by allella** [comment]
@ScottBrenner how are you submitting those diffs / suggestions? Are you checking out the PR and pushing changes to the PR branch?  I'm new here :)

**Comment by ScottBrenner** [comment]
@allella See [step 6 here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review) for details on how to create them (as a PR reviewer) and [here for committing suggested changes](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) (as a PR author)!

**Comment by allella** [comment]
@xarielx looking better.  I think the [] and () may be reversed because the typescriptlang.org URL is visible in this latest change.

File: README.md [hunk 4]
```diff
@@ -29,11 +29,14 @@ To better communicate and more easily build an API and UI, we've decided on a co
...
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
-- [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
-- [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
-- A [React](https://reactjs.org/) frontend using JavaScript/TypeScript and CSS (not Sass)
-- [Next.js](https://nextjs.org/) for server-side rendering
+* [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
+* [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
+* [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
+* A [React](https://reactjs.org) frontend using:
+    * [JavaScript/TypeScript] (https://www.typescriptlang.org/index.html#download-links)
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
    * [JavaScript/TypeScript](https://www.typescriptlang.org/index.html#download-links)
```

This space breaks the hyperlink.

**Comment by xarielx** [comment]
> This space breaks the hyperlink.

Thanks!

**Comment by ScottBrenner** [comment]
Remember to squash when merging into master, please!

**Comment by xarielx** [comment]
I apologize for that @ScottBrenner, you're completely right, that will never happen again, I assumed it was squashing by default.

**Comment by nik-john** [comment]
@all-contributors please add @xarielx   for doc

**Comment by nik-john** [comment]
@all-contributors please add @xarielx for doc

**Pull request merged**